spec/additional-sample-data/valid/fr-bodacc-bilan-01.json

```diff
@@ -0,0 +1,47 @@
+{
+    "issue": {
+        "publication": {
+            "publisher": {
+                "name": "Direction de l'information légale et administrative",
+                "url": "http://www.dila.premier-ministre.gouv.fr/"
+            },
+            "jurisdiction_code": "fr",
+            "title": "Bulletin officiel des annonces civiles et commerciales",
+            "url": "http://www.bodacc.fr/"
+        },
+        "identifier": "20150001",
+        "edition_id": "C",
+        "url": "ftp://echanges.dila.gouv.fr/BODACC/2015/BILAN_BXC20150001.taz"
+    },
+    "date_published": "2015-01-05",
+    "source_url": "ftp://echanges.dila.gouv.fr/BODACC/2015/BILAN_BXC20150001.taz",
+    "sample_date": "2016-01-06T16:05:46Z",
+    "retrieved_at": "2016-01-06T16:05:46Z",
+    "confidence": "HIGH",
+    "other_attributes": {},
+    "publisher": {
+        "name": "GREFFE DU TRIBUNAL D'INSTANCE DE SARREGUEMINES",
+        "identifier": "57"
+    },
+    "uid": "BDC15000100111A",
+    "identifier": 1,
+    "subjects": [{
+        "entity_type": "company",
+        "entity_properties": {
+            "company_number": "303 514 160",
+            "name": "GARAGE POINSIGNON",
+            "company_type": "S.A.R.L.",
+            "registered_address": {
+                "street_address": "zone industrielle, rue Gutenberg",
+                "locality": "Sarreguemines",
+                "postal_code": "57200",
+                "country": "France",
+                "country_code": "FR"
+            },
+            "filings": [{
```

Comment:
Seems reasonable to put the filing on the company (it is a filing of the company after all).